UnityProject/Assets/Scripts/Weapons/FireLemon.cs

```diff
@@ -0,0 +1,221 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Light2D;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using Mirror;
+
+/// <summary>
+/// Generic grenade base.
+/// </summary>
+[RequireComponent(typeof(Pickupable))]
+public class FireLemon : NetworkBehaviour, IPredictedInteractable<HandActivate>, IServerDespawn
+{
+	[Tooltip("Lowest potency explosion")]
+	public Explosion explosionPrefab1;
```

Comment:
Why is it public? Looks like a private variable that isn't referenced outside the script

Reply:
It public because its set in editor, but i would change it to private and [SerialisedField]